core-api/src/main/java/com/optimizely/ab/event/internal/EventFactory.java

```diff
@@ -218,4 +217,19 @@ public LogEvent createConversionEvent(@Nonnull ProjectConfig projectConfig,
 
         return attributesList;
     }
+
+    private static boolean isValidNumber(Object conditionValue) {
```

Comment:
https://github.com/optimizely/java-sdk/pull/249/files#diff-922db348b1c4d7acd0c13f647f829eb6R73

we have two methods with same name, same functionality, can't it go into single helper method.

Reply:
Please add documentation of this method.